AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -23,6 +23,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+adlice.com##div.fc-consent-root:nth-child(19) > div.fc-dialog-container:nth-child(2) > div.fc-dialog.fc-choice-dialog:nth-child(2)
```

Comment:
It's a cookie notice.
Please find similar rule in https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_specific.txt
and add domain to these rules (domain should be added to two rules, one to hide cookie notice, second to unblock scroll).